tests/test_wog.py

```diff
@@ -219,6 +219,14 @@ def test_silithid_swarmer():
 	assert not silithid.can_attack()
 
 
+def test_tentacles_for_arms():
+	game = prepare_game()
+	game.player1.discard_hand()
+	game.player1.give("OG_033").play()
+	game.player1.give(LIGHTS_JUSTICE).play()
+	assert game.player1.hand == ["OG_033"]
+
```

Comment:
Sure. Are we making sure that it winds up in the graveyard instead of infinite looping?